inference-engine/thirdparty/clDNN/src/graph_optimizer/prepare_quantization.cpp

```diff
@@ -41,6 +43,57 @@ bool check_binarization(memory::ptr mem_input_low, memory::ptr mem_input_high, p
 
 void  prepare_quantization::prepare_scale_shift_opt(program &p, quantize_node& quantize_node) {
     const auto& stream = p.get_stream();
+
```

Comment:
What about unit tests for this changes? Isn't it necessary?

Reply:
Maybe later. I tested this case on full network scope in dlbenchmark.